code/modules/research/machinery/pointgiver.dm

```diff
@@ -0,0 +1,16 @@
+/obj/machinery/rnd/pointgiver
+  name = "virtual technology node"
+  desc = "A virtual representation of stored research, ideas and knowledge. This one belongs to [factionname] and will give [reward] points if destoyed."
```

Comment:
I doubt squared parenthesis values will do the trick here.

You should either set it up on `Initialize()`, or add this to its `examine()` :
```
  to_chat(user, <span class='notice'>This one belongs to [factionname] and will give [reward] points if destoyed.</span>)
```